db/migrate/20150625164705_make_merged_at_datetime.rb

```diff
@@ -0,0 +1,6 @@
+class MakeMergedAtDatetime < ActiveRecord::Migration
+  def change
+    remove_column :hubstats_pull_requests, :merged_at
+    add_column :hubstats_pull_requests, :merged_at, :datetime
```

Comment:
All previous ones, we might have to resend from github. I'm not exactly sure how those'll work.